Update dependencies

mint-thompson opened the pull request:
**Description:**
Add eslint config file based on requirements for updated eslint dependencies.

Remove @types/tar dependency because types are now included in the tar dependency.

Minor usage updates for tar and commander based on changes to package export.

Minor fixes for linting rules.

**Testing Instructions:**
Confirm that all tests pass. Confirm that CLI works as expected. There will be additional review and testing when the `new-fpl` branch is being pulled into `main`.

**Related Issue:**
Fixes #27

mint-thompson committed:
Update dependencies

Add eslint config file based on requirements for updated eslint
dependencies.

Remove @types/tar dependency because types are now included in the tar
dependency.

Minor usage updates for tar and commander based on changes to package
export.

Minor fixes for linting rules.

jafeltra commented on package.json:
I can't comment on the line itself, but for `@types/node`, even though we aren't going to update to the v22.x versions, I think we can update to the latest v20.x version.

mint-thompson replied:
Good idea! I've updated that package in the latest commit.

jafeltra reviewed (commented):
This looks good! I just left one small comment.

mint-thompson committed:
Update node types to latest 20.x version

Other packages with recent minor updates are also updated. No change to
the packages in dependency notes.

jafeltra reviewed (approved):
Looks good!

cmoesel reviewed (approved):
Thanks Mint!
